service/matching/task_writer.go

```diff
@@ -248,3 +254,11 @@ func (w *taskWriter) allocTaskIDBlock(prevBlockEnd int64) (taskIDBlock, error) {
 	}
 	return rangeIDToTaskIDBlock(state.rangeID, w.config.RangeSize), nil
 }
+
+// getTaskIDBlock returns the latest taskIDBlock. Safe to be called concurrently.
+func (w *taskWriter) getTaskIDBlock() taskIDBlock {
```

Comment:
the names don't match between old/pri.. should we name this currentTaskIDBlock, or rename the priTaskWriter version to getTaskIDBlock?

Reply:
Good catch; I went back and forth on some of the naming. 👍